.github/workflows/pr-container-build.yaml

```diff
@@ -23,9 +23,21 @@ on:
   pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
     branches: 
-    - main
-    - rhdh-1.[0-9]+
-    - 1.[0-9]+.x
+      - main
+      - rhdh-1.[0-9]+
+      - 1.[0-9]+.x
+    paths:
```

Comment:
Prow marked `PR Publish` as a required check in the repo settings. The problem is that this could cause the workflow to remain in a `Pending` state, blocking it from being merged. See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks.
I think this will also block Prow from merging it (unless we manually override it). I experienced this in the past when the job name did not match the Prow configuration for example: Prow was waiting for a required check that never happened.
How about using a `needs-publish` job that computes the files changed and outputs whether or not images should be built (based on the list below)? Then the "pr-build", which would depend on "needs-publish" could be conditionally skipped if not..